### PR TITLE
Add Translations for zh-rCN

### PR DESCRIPTION
**Pull request by binsee**
Add Translations for zh-rCN

### CODE REVIEW

File: telecine/src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+  <string name="app_name" translatable="false">Telecine</string>
+  <string name="launcher_name" translatable="false">@string/app_name</string>
```
Comment (JakeWharton) [inline telecine/src/main/res/values-zh-rCN/strings.xml]: Delete these two strings. They are not meant to be translated and should only be defined in the `values/strings.xml`.

**Comment by JakeWharton** [comment]
Looks good, just one little thing that needs fixed before merge.

**Comment by binsee** [comment]
OK.

**Comment by binsee** [comment]
Fixed Translations for zh-rCN
Add Translations for zh-rTW,zh-rHK

**Comment by JakeWharton** [comment]
I squashed these into a single commit and merged to master. Thanks!

**Pull request closed without merging**